### PR TITLE
Add defaut-hub command.

### PR DESCRIPTION
**Pull request by mathetake**
Signed-off-by: Takeshi Yoneda <takeshi@tetrate.io>

### CODE REVIEW

File: cmd/default_hub.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	)
+	cmd := &cobra.Command{
+		Use:   "default-hub",
+		Short: `Set or Show the default hub passed to "getistio istioctl install" via "--set hub=" e.g. docker.io/istio`,
+		Long:  `Set or Show the default hub (root for Istio docker image paths) passed to "getistio istioctl install" via "--set hub="  e.g. docker.io/istio`,
+		Example: `# Set the default hub to docker.io/istio
+$ getistio default-hub --set docker.io/istio
+
+# Show the current default hub
+$ getistio default-hub --show
```
Comment (deva26) [inline cmd/default_hub.go:39]: Nice.

File: cmd/default_hub.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+		PreRunE: func(cmd *cobra.Command, args []string) error { return defaultHubCheckFlags(setFlag, showFlag) },
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if setFlag != "" {
+				return defaultHubHandleSet(homedir, setFlag)
+			}
+			defaultHubHandleShow(getistio.GetActiveConfig().DefaultHub)
+			return nil
+		},
+	}
+	cmd.Flags().StringVar(&setFlag, "set", "", "pass the URL of hub, e.g. --set gcr.io/istio-testing")
```
Comment (deva26) [inline cmd/default_hub.go:50]: should we call it Location instead of URL? WDYT?

**Review by deva26 — APPROVED** [review]
Thanks a lot.

File: cmd/default_hub.go (hunk 2 above)
Comment (mathetake) [reply cmd/default_hub.go]: yeah location would be better. fixed.

**Pull request merged**